README.md

```diff
@@ -1,70 +1,45 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Calculator App
+A simple app that displays a calculator using React.
 
-## Available Scripts
 
-In the project directory, you can run:
+## Built With
+- HTML, JavaScript, CSS
+- React
+- Gitflow
 
-### `npm start`
+## Live Demo
+- [Demo](https://anagudelogu.github.io/mathemagics/)
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Getting Started
+> To get a local copy up and running follow these simple example steps.
 
-### `npm test`
+- Clone the repo with this link https://github.com/Danie12345/mathemagics.git
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+- Create a branch to work on your features
 
-### `npm run build`
+- Make a pull request with your changes once you're done
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+## Author: Daniel Malo
+👤 [Github](https://github.com/Danie12345)
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+👤 [Linkedin](https://www.linkedin.com/in/daniel-malo-75218a192/)
 
-### `npm run eject`
+👤 [Twitter](https://twitter.com/DanielMalo_v4)
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+## 🤝 Contributing
+Contributions, issues, and feature requests are welcome!
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+Feel free to check the [issues](https://github.com/anagudelogu/capstone-two/issues) page.
```

Comment:
- [ ] Great job, 👏🏻 but your README file needs to be more descriptive. This is a Math Magicians project, not a Capstone-Two project, and in fact, your issue link takes me to the wrong place. Please fix this.